commands/hypixelstats.js

```diff
@@ -17,46 +17,47 @@ module.exports = {
     cooldown: 60, // feel free to change it
     channelWhiteList: ["549894938712866816", "624881429834366986", "730042612647723058", "749391414600925335"], //bots staff-bots bot-dev bot-translators
     allowDM: true,
-    execute (strings, message, args) {
+    execute(strings, message, args) {
         const executedBy = strings.executedBy.replace("%%user%%", message.author.tag)
         const madeBy = strings.madeBy.replace("%%QkeleQ10%%", "QkeleQ10#6046")
         let username = args[0]
 
         // make a response to the slothpixel api (hypixel api but we dont need an api key)
-        fetch(`https://api.slothpixel.me/api/players/${username}`, {method: "Get"})
+        fetch(`https://api.slothpixel.me/api/players/${username}`, { method: "Get" })
             .then(res => (res.json())) // get the response json
             .then((json) => { // here we do stuff with the json
-                if (json.error === "Player does not exist") {
-                    throw strings.hypixelApiError
-                }
+
+                if (json.error === "Player does not exist") throw "falseUser"
                 else if (json.error !== undefined) { // if other error we didn't plan for appeared
                     console.log(`Welp, we didn't plan for this to happen. While you have a mental breakdown, enjoy this little error I have for you\n${json.error}`)
+                    throw "apiError"
                 }
+
                 let rank // some ranks are just prefixes so this code accounts for that
-                if (json.prefix !== null) {
-                    rank = json.prefix.replace(/&([1-9]|[a-z])/g, "")
-                }
-                else {
-                    rank = json.rank_formatted.replace(/&([1-9]|[a-z])/g, "")
-                }
+                if (json.prefix !== null) rank = json.prefix.replace(/&([1-9]|[a-z])/g, "")
+                else rank = json.rank_formatted.replace(/&([1-9]|[a-z])/g, "").replace("_PLUS", "+")
```

Comment:
The `.replace("_PLUS", "+")` is redundant since we're using rank_formatted and not rank

Reply:
ahhhhhhhhhhhhhhhhh okay thank